akka-http2-support/src/main/scala/akka/http/scaladsl/model/http2/Http2StreamIdHeader.scala

```diff
@@ -6,6 +6,7 @@ package akka.http.scaladsl.model.http2
 
 import akka.http.impl.engine.ws.InternalCustomHeader
 
+// TODO should the streamId not be a Long? I may be mis-remembering the spec
```

Comment:
I think they took 31 bits by purpose because that still fits into a signed 32 bit integer.